plaidml/edsl/edsl.h

```diff
@@ -805,8 +811,10 @@ enum class SortDirection {
 /// \param direction SortDirection
 /// \return Tensor
 ///
-inline Tensor argsort(const Tensor& x, int axis = -1, SortDirection direction = SortDirection::ASC) {
-  return intrinsic("argsort", x, static_cast<int64_t>(axis), static_cast<int64_t>(direction));
+inline Tensor argsort(const Tensor& x, int axis = -1, SortDirection direction = SortDirection::ASC,
+                      SortStability stability = SortStability::STABLE) {
```

Comment:
Based on both TensorFlow's approach and my own intuition, I think we should instead default to the (probably more performant) unstable sort and require the user to explicitly request a stable sort.